src/lib.rs

```diff
@@ -72,7 +72,7 @@ mod crypto;
 mod dns;
 mod kdc;
 mod krb;
-mod rustls;
+pub mod rustls;
```

Comment:
issue: Let’s keep this module private. Following previous advice, re-export from lib.rs and ensure the item is properly hidden from the documentation.

Reply:
done